src/routes/ConsortiumManager/settings/users/general/PermissionSets/hooks/useSelectPermissionSetById.js

```diff
@@ -0,0 +1,18 @@
+import { useTenantPermissions } from '../../../../../../../hooks';
+
+export const useSelectPermissionSetById = ({ permissionSetId, tenantId }) => {
```

Comment:
at first this hook doesn't select anything, it should be `usePermissionsSet`, secondly why not to fetch by id instead of fetching list and searching by id in memory?

https://github.com/folio-org/mod-permissions/blob/master/descriptors/ModuleDescriptor-template.json#L87

Reply:
I have used this approach but the response did not satisfy the Accordion component. 
It returns subPermissions as string of arrays `subPermissions: ['ui.permission']` but it was required as an object and all the data of the permissions. I have consulted Tatsiana Tarhonskaya and Yury. Only This option worked for me.
```
{ subPermissions: ["ui-agreements.agreements.delete",]}

expected { subPermissions: [{

          "permissionName": "ui-consortia-settings.consortium-manager.view",

          "displayName": "Consortium manager: Can view existing settings",

          "id": "31de539c-64db-4f4d-8325-0dd57452ab9d",

          "tags": [],

          "subPermissions": [

              "module.consortia-settings.enabled"

          ],

          "childOf": [

              "ui-consortia-settings.consortium-manager.edit",

              "ui-consortia-settings.consortium-manager.share"

          ], ]}
```

Reply:
@NikitaSedyx I have tried this parmas but it din't resolve the issue above. If you know something it would be a great improvement using fetch by id.
`searchParams: {
      expandSubs: true,
    },`

Reply:
seems BE doesn't support....

in this case
1. rename hook to usePermissionsSet
2. put id to query params so you don not fetch everything and filter by id on client side, instead you filter by id on BE side

Reply:
query: `mutable==true&id=${id}`,